packages/relic/lib/src/headers/util/headers_util.dart

```diff
@@ -0,0 +1,10 @@
+part of '../../headers.dart';
+
+/// Helper method to check if a string is valid Base64.
+///
+/// This method is used internally to validate that the Base64-encoded credentials
+/// are correctly formatted before attempting to decode them.
+bool _isValidBase64(String str) {
```

Comment:
Isn't this better as an extension to a `String`. Also, no need to make it private, this can be useful as a part of the public interface.